plugins/blur/blur-base.cpp

```diff
@@ -232,6 +244,7 @@ void wf_blur_base::render(wf::texture_t src_tex, wlr_box src_box,
     /* Blend blurred background with window texture src_tex */
     blend_program.uniformMatrix4f("mvp", glm::inverse(target_fb.transform));
     /* XXX: core should give us the number of texture units used */
+    blend_program.uniform1f("sat", saturation_opt);
```

Comment:
please move this before the comment, because the comment is related to the next line

Reply:
Ah, nice catch.